### ISSUE TITLE
Disabling cache flushes settings

### ISSUE THREAD

**Issue by kraftbj** [Automattic/wp-super-cache#33]
Unconfirmed via https://wordpress.org/support/topic/disabling-and-reenabling-the-cache-should-not-clear-all-my-settings?replies=1

**Comment by jnylin** [comment]
I can confirm this. If I disable caching under the Easy-tab, the settings under Advanced is cleared, but only the first section.

Another strange behaviour, maybe related, is that if I change a setting in the first section under Advanced my settings about expiry time is cleared.

**Comment by jnylin** [comment]
A relevant row: https://github.com/Automattic/wp-super-cache/blob/master/wp-cache.php#L450

**Comment by jnylin** [comment]
And here is a relevant row for the other strange behaviour I descriped: https://github.com/Automattic/wp-super-cache/blob/master/wp-cache.php#L1055

**Comment by jnylin** [comment]
I forked the repo so I when I get time I will try to fix it.

**Comment by jnylin** [comment]
I created a pull request for the problem with cleared expiry time if you change settings. When I wrote this fix I also looked at the original problem with flushed settings and realised that it is that if you disable caching under the Easy tab it flushes all settings under the Advanced tab, if you disable caching under the Advanced tab the settings remain. I therefore think it's not a bug but a question about the admin interface.
